README.md

```diff
@@ -138,6 +166,8 @@ From the local path of the sensu-influxdb-handler repository:
 dep ensure
 ```
 
+Note: Sensu-influxdb-handler also uses go modules in case you're not using dep.
+
```

Comment:
Thanks for converting the project to modules. It doesn't make sense to support dep _and_ modules at the same time, as we need to pick one or the other for the CI pipeline, so there's some other cleanup required to fully migrate the project to a module. This includes removing the Gopkg.lock and Gopkg.toml files, removing `dep` from the travis file, and updating the readme. If you'd like some guidance on this, please let me know! Otherwise, I can carry those changes over.

Reply:
Actually I realized the go.mod is referencing my repo. I think this would have to be updated to this repo? Yes, guidance here would be great. Is there an example plugin I can look at to see how it's being done with go modules in Travis CI?

Reply:
Well it looks like [gometalinter](https://github.com/alecthomas/gometalinter/tree/v1.2.1) does not support modules, and is deprecated in favour of [golangci-lint](https://github.com/golangci/golangci-lint). This is causing the CI builds to fail. Should we move towards golangci-lint or adopt the GitHub actions that other handler projects are using?